lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -514,11 +540,10 @@ if ($state -in ("absent", "reinstalled")) {
 if ($state -in ("downgrade", "latest", "present", "reinstalled")) {
 
     Choco-Install -package $package -version $version -force $force -timeout $timeout `
-        -skipscripts $skipscripts -source $source -installargs $installargs `
-        -packageparams $packageparams -allowemptychecksums $allowemptychecksums `
-        -ignorechecksums $ignorechecksums -ignoredependencies $ignoredependencies `
-        -allowdowngrade ($state -eq "downgrade") -proxy_url $proxy_url `
-        -proxy_username $proxy_username -proxy_password $proxy_password `
+        -skipscripts $skipscripts -source $source -installargs $installargs -packageparams $packageparams `
```

Comment:
Same here, keep the line the same but just add `-source_username` and `-source_password` onto the end. No need to change the ordering.

Reply:
Adding it on the end breaks shippable.
So i removed it because it is defined in the options or am i wrong?

I'll revert back to the original